packages/veritone-widgets/src/widgets/FilePicker/index.js

```diff
@@ -83,7 +85,9 @@ class FilePicker extends React.Component {
     onPick: noop,
     percentByFiles: [],
     height: 450,
-    width: 600
+    width: 600,
+    enableResize: false,
+    aspectRatio: 16/9,
```

Comment:
Would square be a better default here? It's mostly for avatars right?

Reply:
yeah. I will change it to 1/1.